### PR TITLE
Added ONNX support

### PR DESCRIPTION
**Pull request by antas-marcin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ COPY requirements.txt .
 RUN pip3 install -r requirements.txt
```
Comment (dirkkul) [inline Dockerfile:6]: update docker file?

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      LOCAL_REPO: sum-transformers
+      REMOTE_REPO: semitechnologies/sum-transformers
+      MODEL_NAME: ${{matrix.model_name}}
+      MODEL_TAG_NAME: ${{matrix.model_tag_name}}
+      ONNX_RUNTIME: ${{matrix.onnx_runtime}}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
```
Comment (dirkkul) [inline .github/workflows/main.yaml:43]: when you update the docker file, change to 3.10/11 here (depending on the base image)

File: test/smoke_test_facebook-bart-large-cnn.py [hunk 3]
```diff
@@ -53,6 +54,7 @@ def testSum(self):
         self.assertEqual(req_body['text'], res_body['text'])
         self.assertTrue(str(res_body['summary']).find("Weaviate"))
         self.assertGreater(len(str(res_body['summary']).strip()), 0)
+        print(f"Test 2 result: {str(res_body['summary'][0]['result'])}")
```
Comment (dirkkul) [inline test/smoke_test_facebook-bart-large-cnn.py:57]: remove prints again?

**Review by dirkkul — APPROVED** [review]

File: test/smoke_test_facebook-bart-large-cnn.py (hunk 3 above)
Comment (antas-marcin) [reply test/smoke_test_facebook-bart-large-cnn.py:57]: I will remove those prints in the next PR

File: .github/workflows/main.yaml (hunk 2 above)
Comment (antas-marcin) [reply .github/workflows/main.yaml:43]: I will harden the docker image in the next PR

**Pull request merged**